client/state/user-settings/helpers.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
FWIW these could be used in other places. I wonder if we could put them at a more general place in `/lib` where they could make broader use.

Reply:
True. I'll make a note to follow up after the reduxification 👍🏻